superset/translations/pt_BR/LC_MESSAGES/messages.po

```diff
@@ -846,7 +846,7 @@ msgstr "Escolha um tipo de visualização"
 
 #: superset/assets/javascripts/addSlice/AddSliceContainer.jsx:87
 msgid "Create new slice"
-msgstr "Crie uma nova fatia"
+msgstr "Crie uma nova pedaço"
```

Comment:
I'm not pt/BR more just PT. But this does not sound good to me. At least "Crie um novo pedaço", but I think I prefer "fatia"